api.go

```diff
@@ -281,8 +281,6 @@ func (r *Router) Call(ctx context.Context, bucketID uint64, mode CallMode,
 
 	for {
 		if spent := time.Since(requestStartTime); spent > timeout {
-			r.metrics().RequestDuration(spent, fnc, "", false, false)
```

Comment:
Why we don`t send metric if request done with error?

Reply:
Because we decided to send metrics for particular requests to storage tarantools. This deleted code sends something else: aggregated metrics over several calls to different storage tarantools; also includes sleep time, timings to bucket_set/bucket_reset and etc. Finally, the aggregated timing can be measured by the caller outside of this function.